sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ServiceBusClientBuilderJavaDocCodeSamples.java

```diff
@@ -59,17 +74,20 @@ public void instantiateSenderSync() {
     }
 
     @Test
-    public void instantiateSenderAsync() {
-        // BEGIN: com.azure.messaging.servicebus.sender.async.client.instantiation
-        // Retrieve 'connectionString' and 'topicName' from your configuration.
+    public void instantiateProcessor() {
+        // BEGIN: com.azure.messaging.servicebus.processor.client.instantiation
+        // Retrieve 'connectionString' and 'queueName' from your configuration.
         ServiceBusClientBuilder builder = new ServiceBusClientBuilder()
             .connectionString(connectionString);
-        ServiceBusSenderAsyncClient sender = builder
-            .sender()
-            .topicName(topicName)
-            .buildAsyncClient();
-        // END: com.azure.messaging.servicebus.sender.async.client.instantiation
-        sender.sendMessage(new ServiceBusMessage("payload")).subscribe();
+        ServiceBusProcessorClient processor = builder
```

Comment:
The builder methods can be chained instead.
```java
ServiceBusProcessorClient processor = new ServiceBusClientBuilder()
            .connectionString(connectionString)
            .processor()
            .queueName(queueName)
            .processMessage(System.out::println)
            .processError(context -> System.err.println(context.getErrorSource()))
            .buildProcessorClient();

```

Reply:
In intentionally kept it this way , so it promotes sharing of connection.